content/02-understand-prisma/05-under-the-hood.mdx

```diff
@@ -51,3 +51,87 @@ A Prisma engine is the **direct interface to the database**, any higher-level in
 As an example, Prisma Client connects to the [query engine](../reference/tools-and-interfaces/prisma-client/query-engine) in order to read and write data in a database:
 
 ![](https://i.imgur.com/Nwwx5Cv.png)
+
+### Using custom engine binaries
+
+By default, all binaries are automatically downloaded into the `node_modules\@prisma\cli` folder when you install or update `@prisma\cli`. You might want to use a custom binary file if:
+
+* Automated download of binaries is not possible.
+* You have created your own engine binary (for testing, or for an OS that is not officially supported).
+
+Use the following environment variables to specify custom locations for your binaries:
+
+* `PRISMA_MIGRATION_ENGINE_BINARY` (Migration engine)
+* `PRISMA_QUERY_ENGINE_BINARY` (Query engine)
+* `PRISMA_INTROSPECTION_ENGINE_BINARY` (Introspection engine)
+* `PRISMA_FMT_BINARY` (for `npx prisma format`)
+
+> **Tip**: The  environment allows you to use a c
```

Comment:
I guess you wanted to finish with something here :)

Reply:
The environment allows you to use a c....old-freezer to store the corpses!